index.html

```diff
@@ -41,9 +41,127 @@ <h2 class="green">I’m a software developer</h2>
         <div class="next-section">
             <img class="next-section-svg" src="assets/icons/round-arrow-down.svg" alt="">
         </div>
+        <div class="background-image-wrapper">
+            <img id="into-background" src="assets/images/background-mobile.svg" alt="">
+        </div>
         
-        <img id="into-background" src="assets/images/background-mobile.svg" alt="">
-        
+    </section>
+    <section class="recent-works">
+        <h2 class="section-title">My Recent Works</h2>
+        <div class="indicator"></div>
+        <div class="projects">
+            <div class="project-card">
+                <div class="project-info">
+                    <div class="project-title">
+                        Multi-Post Stories Gain+Glory
+                    </div>
+                    <div class="project-stacks">
+                        <div>Ruby on Rails</div>
+                        <div>css</div>
+                        <div>JavaScript</div>
+                        <div>html</div>
+                    </div>
+                    <div class="button-wrapper">
+                        <a  href="#" class="button">See Project</a> 
+                    </div>
+                </div>
+            </div>
+            <div class="project-card">
+                <div class="project-info">
+                    <div class="project-title">
+                        Multi-Post Stories Gain+Glory
+                    </div>
+                    <div class="project-stacks">
+                        <div>Ruby on Rails</div>
+                        <div>css</div>
+                        <div>JavaScript</div>
+                        <div>html</div>
+                    </div>
+                    <div class="button-wrapper">
+                        <a  href="#" class="button">See Project</a> 
+                    </div>
```

Comment:
Could we implement the button interaction for our chosen button color scheme as given by the design?

Here are the interactions given:
![image](https://user-images.githubusercontent.com/20554583/186877797-a2ba99f6-72be-4df4-8b91-b2cdde5f77ee.png)

Reply:
got it, added hover and active interactions